privacy-profile-composer/pkg/composer/grpc_client.go

```diff
@@ -75,3 +76,50 @@ func Run_client() {
 	}
 	log.Println(profile)
 }
+
+func sendComposedProfile(fqdn string, purpose string, piiTypes []string, thirdParties []string) api.StatusType {
+	var (
+		composerSvcAddr = flag.String("addr", "http://prose-server.prose-system.svc.cluster.local:50051", "the address to connect to")
```

Comment:
also, if we are calling this function from golang filter, i don't think we get to have cli flags.

Reply:
See my response to the previous comment on this function. I've added a todo to handle it later, when I deal with the Jaeger trace querying implementation.